helm/kanister-operator/README.md

```diff
@@ -23,7 +23,7 @@ This chart bootstraps a kanister-operator deployment on a [Kubernetes](http://ku
 To install the chart with the release name `my-release`:
 
 ```console
-$ helm install --name my-release kanister/kanister-operator
+$ helm install my-release kanister/kanister-operator
```

Comment:
I think we should add `kubectl create namespace kanister` and `--namespace kanister` here

Reply:
Also add `--namespace kanister` to the delete command at line 38

Reply:
Or just add the `--create-namespace` option to let Helm auto-create it.